### DISCUSSION TITLE
Adding a column to the table

### DISCUSSION THREAD

**Discussion by DorijanH** [ShapeCrawler/ShapeCrawler#650] [Q&A] [answered]
Hi there,

is there a way to manipulate the columns in a table, for example adding a new column or duplicating an existing one? Something similar to the `Duplicate` method available in the `ITableRow` interface

**Comment by AdamShakhabov** [comment, marked as answer]
Hello,

no, there is not such feature yet.

**Reply by AdamShakhabov** [reply]
The issue https://github.com/ShapeCrawler/ShapeCrawler/issues/661 was created to implement this feature.
